lib/KeyboardAwareFlatList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import createReactClass from 'create-react-class';
+import PropTypes from 'prop-types';
+import { FlatList } from 'react-native';
+import KeyboardAwareMixin from './KeyboardAwareMixin';
+
+const KeyboardAwareFlatList = createReactClass({
+  propTypes: {
+    viewIsInsideTabBar: PropTypes.bool,
+    resetScrollToCoords: PropTypes.shape({
+      x: PropTypes.number,
+      y: PropTypes.number,
+    }),
+  },
+  mixins: [KeyboardAwareMixin],
+
+  componentWillMount: function () {
+    this.setViewIsInsideTabBar(this.props.viewIsInsideTabBar)
+    this.setResetScrollToCoords(this.props.resetScrollToCoords)
+  },
+
+  render: function () {
+    return (
+      <FlatList
+        ref='_rnkasv_keyboardView'
```

Comment:
strings are used in the existing KeyboardAwareScrollView and KeyboardAwareListView components. KeyboardAwareMixin also references ref as `this.refs._rnkasv_keyboardView`, which I think should be `this._rnkasv_keyboardView` for callback refs. Do you want me to update everywhere to use callback refs in this PR? Or, leave it as is, and open another PR to switch to callback refs?